pkg/minikube/constants/constants.go

```diff
@@ -69,70 +68,24 @@ var KubeconfigPath = clientcmd.RecommendedHomeFile
 // KubeconfigEnvVar is the env var to check for the Kubernetes client config
 var KubeconfigEnvVar = clientcmd.RecommendedConfigPathEnvVar
 
-// MinikubeContext is the kubeconfig context name used for minikube
-const MinikubeContext = "minikube"
-
-// MinikubeEnvPrefix is the prefix for the environmental variables
-const MinikubeEnvPrefix = "MINIKUBE"
-
 // DefaultMachineName is the default name for the VM
 const DefaultMachineName = "minikube"
 
 // DefaultNodeName is the default name for the kubeadm node within the VM
 const DefaultNodeName = "minikube"
 
-// DefaultStorageClassProvisioner is the name of the default storage class provisioner
-const DefaultStorageClassProvisioner = "standard"
-
-// Cache is used to modify the cache field in the config file
-const Cache = "cache"
-
 // MountProcessFileName is the filename of the mount process
 var MountProcessFileName = ".mount-process"
 
 const (
-	// DefaultEmbedCerts  is if the certs should be embedded in the kubeconfig file
-	DefaultEmbedCerts = false
-	// DefaultKeepContext is if we should keep context by default
-	DefaultKeepContext = false
 	// SHASuffix is the suffix of a SHA-256 checksum file
 	SHASuffix = ".sha256"
-	// DefaultMemorySize is the default memory which will be allocated to minikube, in megabytes
-	DefaultMemorySize = "2000mb"
-	// MinimumMemorySize is the minimum memory size, in megabytes
-	MinimumMemorySize = "1024mb"
-	// DefaultCPUS is the default number of cpus of a host
-	DefaultCPUS = 2
-	// MinimumCPUS is the minimum number of cpus of a host
-	MinimumCPUS = 2
-	// DefaultDiskSize is the default disk image size, in megabytes
-	DefaultDiskSize = "20000mb"
-	// MinimumDiskSize is the minimum disk image size, in megabytes
-	MinimumDiskSize = "2000mb"
-	// DefaultVMDriver is the default virtual machine driver name
-	DefaultVMDriver = DriverVirtualbox
-	// DefaultStatusFormat is the default format of a host
-	DefaultStatusFormat = `host: {{.Host}}
-kubelet: {{.Kubelet}}
-apiserver: {{.APIServer}}
-kubectl: {{.Kubeconfig}}
-`
-	// DefaultAddonListFormat is the default format of addon list
-	DefaultAddonListFormat = "- {{.AddonName}}: {{.AddonStatus}}\n"
-	// DefaultConfigViewFormat is the default format of config view
-	DefaultConfigViewFormat = "- {{.ConfigKey}}: {{.ConfigValue}}\n"
-	// DefaultCacheListFormat is the default format of cache list
-	DefaultCacheListFormat = "{{.CacheImage}}\n"
 	// GithubMinikubeReleasesURL is the URL of the minikube github releases JSON file
 	GithubMinikubeReleasesURL = "https://storage.googleapis.com/minikube/releases.json"
```

Comment:
This can be moved to `pkg/minikube/notify`, as all uses are calls to functions within it.

Reply:
done